scrapi/harvesters/addis_ababa.py

```diff
@@ -6,6 +6,27 @@
 from __future__ import unicode_literals
 
 from scrapi.base import OAIHarvester
+from scrapi.base import helpers
+
+
+def format_uris_addis_ababa(*args):
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
```

Comment:
Is there ever more than one `provider_uri`? If there is, you are overwriting this variable below, which may result in information loss.